packages/fast-colors/src/named-colors.ts

```diff
@@ -0,0 +1,894 @@
+import { ColorRGBA64Config } from "./color-rgba-64";
+export type NamedColors =
```

Comment:
can we get a documentation link to where these colors are defined, i assume mdn has these listed

Reply:
added